src/disease/schemas.py

```diff
@@ -55,72 +56,47 @@ class NamespacePrefix(Enum):
     ICD10CM = "icd10.cm"
-    ICD11 = "icd11"
     ICDO = "icdo"
-    IDO = "ido"
     IMDRF = "imdrf"
     KEGG = "kegg.disease"
     MEDDRA = "meddra"
     MEDGEN = "medgen"
     MESH = "mesh"
-    MF = "mf"
-    MP = "MP"
-    MPATH = "mpath"
-    NIFSTD = "nifstd"
-    OBI = "obi"
-    OGMS = "ogms"
     ORPHANET = "orphanet"
-    PATO = "pato"
-    SCDO = "scdo"
     UMLS = "umls"
-    WIKIPEDIA = "wikipedia.en"
-    WIKIDATA = "wikidata"
-
-
-# Source to URI. Will use OBO Foundry persistent URL (PURL) or source homepage
-NAMESPACE_TO_SYSTEM_URI: dict[NamespacePrefix, str] = {
-    NamespacePrefix.NCIT: "http://purl.obolibrary.org/obo/ncit.owl",
-    NamespacePrefix.MONDO: "http://purl.obolibrary.org/obo/mondo.owl",
-    NamespacePrefix.DO: "http://purl.obolibrary.org/obo/doid.owl",
-    NamespacePrefix.DOID: "http://purl.obolibrary.org/obo/doid.owl",
-    NamespacePrefix.OMIM: "https://www.omim.org",
-    NamespacePrefix.ONCOTREE: "https://oncotree.mskcc.org",
-    NamespacePrefix.COHD: "https://cohd.io",
-    NamespacePrefix.DECIPHER: "https://www.deciphergenomics.org",
-    NamespacePrefix.EFO: "https://www.ebi.ac.uk/efo/",
-    NamespacePrefix.GARD: "https://rarediseases.info.nih.gov",
-    NamespacePrefix.HP: "http://purl.obolibrary.org/obo/hp.owl",
-    NamespacePrefix.HPO: "http://purl.obolibrary.org/obo/hp.owl",
-    NamespacePrefix.ICD11: "https://icd.who.int/en/",
-    NamespacePrefix.ICDO: "https://www.who.int/standards/classifications/other-classifications/international-classification-of-diseases-for-oncology/",
-    NamespacePrefix.KEGG: "https://www.genome.jp/kegg/disease/",
-    NamespacePrefix.MEDDRA: "https://www.meddra.org",
-    NamespacePrefix.MEDGEN: "https://www.ncbi.nlm.nih.gov/medgen/",
-    NamespacePrefix.MESH: "https://id.nlm.nih.gov/mesh/",
-    NamespacePrefix.MP: "http://purl.obolibrary.org/obo/mp.owl",
-    NamespacePrefix.OBI: "http://purl.obolibrary.org/obo/obi.owl",
-    NamespacePrefix.ORPHANET: "https://www.orpha.net",
-    NamespacePrefix.PATO: "http://purl.obolibrary.org/obo/pato.owl",
-    NamespacePrefix.UMLS: "https://www.nlm.nih.gov/research/umls/index.html",
-    NamespacePrefix.WIKIPEDIA: "https://en.wikipedia.org",
-    NamespacePrefix.WIKIDATA: "https://www.wikidata.org",
-}
-
-# URI to source
-SYSTEM_URI_TO_NAMESPACE = {
-    system_uri: ns.value for ns, system_uri in NAMESPACE_TO_SYSTEM_URI.items()
-}
+
+
+# Source to URI. Will use  system URI prefix, OBO Foundry persistent URL (PURL), or source homepage
+NAMESPACE_TO_SYSTEM_URI: MappingProxyType[NamespacePrefix, str] = MappingProxyType(
+    {
+        NamespacePrefix.NCIT: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&code=",
+        NamespacePrefix.MONDO: "http://purl.obolibrary.org/obo/mondo.owl",
+        NamespacePrefix.DO: "https://disease-ontology.org/?id=",
+        NamespacePrefix.DOID: "https://disease-ontology.org/?id=",
+        NamespacePrefix.OMIM: "https://omim.org/entry/",
```

Comment:
this will work but they recommend their own in-house pURL https://www.omim.org/help/linking#1_9

Reply:
looks like the omim pURL is also what the gene normalizer will be using, would be good to be consistent

Reply:
@jsstevenson I appreciate you checking the values of these!